snakemake/io.py

```diff
@@ -965,7 +981,16 @@ def multiext(prefix, *extensions):
             "Extensions for multiext may not contain path delimiters "
             "(/,\) and must start with '.' (e.g. .txt)."
         )
-    return [flag(prefix + ext, "multiext", flag_value=prefix) for ext in extensions]
+
+    def copy_flags(source, dest):
```

Comment:
I think it's rather helpful to be able to propagate specific flags to individual members of a `multiext` list.